lib/was/thumbnail_service/picker/memento_picker_threshold_grouping.rb

```diff
@@ -2,7 +2,7 @@ module Was
   module ThumbnailService
     module Picker
       class MementoPickerThresholdGrouping
-         @threshold = 0
+        @threshold = Rails.configuration.respond_to?(:threshold) ? Rails.configuration.threshold : 0
```

Comment:
Hi Ahmed,
A few questions I have when looking at this:

1. Does `MementoPickerThresholdGrouping` need to be a class? It doesn't appear that an instance of it is ever initialized AFAICT.
2. Does `@threshold` need to be an instance variable set at the class/module level?  Could it just be another method?(which would also make it much easier to test)